lms/djangoapps/cohort_management_dashboard/tab.py

```diff
@@ -1,11 +1,11 @@
-from django.utils.translation import ugettext_noop
+from django.utils.translation import ugettext
```

Comment:
Can we import `ugettext as _`? It will be very usable for the next time.

Reply:
As for me, better use ugettext instead alias, because edX has separate underscore module for js template rendering. So ugettext is more clear for me.